mission_control/navigator_alarm/navigator_alarm/alarm_handlers/kill.py

```diff
@@ -1,22 +1,21 @@
 #!/usr/bin/env python
 import rospy
 from kill_handling.broadcaster import KillBroadcaster
+from _template import HandlerBase
 
-
-class Handler(object):
+class Handler(HandlerBase):
```

Comment:
The alarm handler script does. I suppose it could - if you give me alittle time I can implement that if you'd like